### ISSUE TITLE
Great extension/add-on

### ISSUE THREAD

**Issue by syverik** [meetDeveloper/Dictionary-Anywhere#23]
Need whitelist the url(s) for my ad block, if possible please list them n would be highly appreciative. By for now

![webster](https://user-images.githubusercontent.com/53056881/87875329-9d579580-c99e-11ea-8cb2-9fae954d4e07.png)

**Comment by meetDeveloper** [comment]
@syverik are you the one with slashdot website problem?

**Comment by syverik** [comment]
Well you are a smart person, and yes I am

**Comment by meetDeveloper** [comment]
@syverik Looking into this. Is this problem only experienced when you turn on adblock?

**Comment by syverik** [comment]
Yes, I recently switched uBlock Origin and have the same problem persists, only on this site(slashdot) that I have noticed so far. If I turn off completely and wait the extra for the ads load then I can see the definitions on slashdot using your amazing, flabbergasting add-on

**Comment by meetDeveloper** [comment]
@syverik Hmm, i noticed that when i turned off the ublock origin and then reload the page, the popup shows, but if I turned on it does not show. You waited for ads to show when you turned off ublock origin without reloading the page right?

**Comment by syverik** [comment]
Correct

**Comment by syverik** [comment]
ublock on = does not work
ublock off = does work

**Comment by meetDeveloper** [comment]
I found the cause, that some css style sheet that ublock origin apply to slashdot makes the display none for my addon.

**Comment by syverik** [comment]
And I'm taking it there is no work around? Appreciate you looking into this, shall still use your add-on for other stuff

**Comment by meetDeveloper** [comment]
There is a way, ideally this should not have happened as I am using shadow DOM so styling from outside should not have bleed in. But somehow this is happening, I need to find out why.

I will look into this closely and promise a fix soon. Sorry for the inconvenience.

**Comment by syverik** [comment]
Much respect for what you are doing, almost like you are reinventing the pencil with this app. Don't know how to say how much easier this makes learning new words for me. Was reading Webster, more recently highlighting and searching on the googol or using my voice for those alexa or nest hubs while reading many a book(of forgotten lore :P) so I don't waste so much finding a word that I've already felt just haven't had the words. So thanks again!!!

**Comment by meetDeveloper** [comment]
@syverik Thanks a lot :)

**Comment by meetDeveloper** [comment]
@syverik Can you try now and let me know, it should now be working. Sorry for very late reply, I fixed these kind of issues few days back and was now trying on this website, and now it was working.
